package.json

```diff
@@ -26,7 +26,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "CI=false && react-scripts build",
```

Comment:
What does `CI=false` do?  Is it setting an environment variable somewhere?

Reply:
I added that part because when running the GitLab CI I get the error:
> Treating warnings as errors because process.env.CI = true.

I got that solution from this discussion: https://github.com/facebook/create-react-app/issues/3657